tasks/java.yml

```diff
@@ -2,6 +2,8 @@
 - name: "Check if Java is installed"
   shell: which java
   register: check_java
+  failed_when: check_java.rc not in  (0, 1)
+  changed_when: False
```

Comment:
for me (vagrant trusty64) `.rc` is still 0 tho even w/ `command -v` if java is not found. regardless `command -v` seems the recommended way to check for existence anyway.